eng/sdl-tsa-vars.config

```diff
@@ -0,0 +1,9 @@
+-TsaInstanceURL $(TsaInstanceURL)
```

Comment:
You need to define actual values for each of the $() variables

Reply:
i have a variable group defining these value. can we use that here ?

Reply:
You need to copy the values from there to here. Also, include `SourceToolsList ` since you are including it [here](https://github.com/dotnet/runtime/blob/master/eng/pipelines/official/stages/publish.yml#L45). You can look at what aspnetcore [did](https://github.com/dotnet/aspnetcore/blob/master/eng/sdl-tsa-vars.config) as an example